backend/app/api/endpoints/pet.py

```
@@ -0,0 +1,62 @@
+from app.api.dependencies import get_db
+from app.core import security
+from app.models.user import User
+from app.schemas.pet import PetResponse, PetUpdate
+from app.services.pet import pet_service
+
+router = APIRouter()
+
+
+@router.get("", response_model=PetResponse)
+async def get_current_user_pet(
+    db: Session = Depends(get_db),
```

Ruff (0.14.14)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)